### ISSUE TITLE
Is the .gitmodules file incomplete?

### ISSUE THREAD

**Issue by brando90** [UCSD-PL/proverbot9001#98]
I was trying to build `constructive-geometry` but it was missing for some reason. When I tried to reinitialize it just wouldn't reinitialize it for some reason, idk why. 

Then I noticed it wasn't even in the `.gitmodules` file. Does this mean that the `.gitmodules` file is missing some git projects or something?

**Comment by brando90** [comment]
but the `.gitignore` file also has ignoring `/coq-projects/`. How are people suppose to have the coq-projects or update them?

**Comment by brando90** [comment]
shouldn't there be 124 or 136 entries in the `.gitignore` file?
```
(metalearning_gpu) brando9~/proverbot9001 $ find $HOME/proverbot9001/coq-projects -maxdepth 1 -type d | wc -l
136
(metalearning_gpu) brando9~/proverbot9001 $ total_num_coq_projs=$(jq length coqgym_projs_splits.json)
(metalearning_gpu) brando9~/proverbot9001 $ echo total_num_coq_projs = $total_num_coq_projs
total_num_coq_projs = 124
```

**Comment by HazardousPeach** [comment]
Yeah originally, the coq projects were committed direclty into the Proverbot repo, instead of using submodules. Projects were converted over to submodules as-needed while I was fixing builds, and it looks like constructive-geometry never made the switch, it's still committed directly into the repo. So, if it's missing for you, you should be able to get it with `git checkout coq-projects/constructive-geometry`. If you'd like to move it to a submodule to match the others, pull requests are welcome!
